src/include/execution/plans/index_scan_plan.h

```diff
@@ -0,0 +1,50 @@
+//===----------------------------------------------------------------------===//
+//
+//                         BusTub
+//
+// index_scan_plan.h
+//
+// Identification: src/include/execution/plans/index_scan_plan.h
+//
+// Copyright (c) 2015-19, Carnegie Mellon University Database Group
```

Comment:
may need to make these 2015-20

Reply:
This range is wrong for almost all files, maybe we gonna change them all at once later